docs/api/classifier/class_balance.rst

```diff
@@ -8,12 +8,14 @@ classification model.
 
 .. code:: python
 
+    from sklearn.model_selection import train_test_split
+
     # Load the classification data set
-    data = load_data('occupancy')
+    data = load_data("occupancy")
 
     # Specify the features of interest and the classes of the target
     features = ["temperature", "relative humidity", "light", "C02", "humidity"]
-    classes = ['unoccupied', 'occupied']
+    classes = ["unoccupied", "occupied"]
```

Comment:
When code blocks contained both single and double quotes, I changed them to double quotes just so it would be consistent.